recipes/pyre/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "pyre" %}
+{% set version = "1.9.5" %}
+{% set config_version = "3.5.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - folder: pyre
+    url: https://github.com/pyre/pyre/archive/v{{ version }}.tar.gz
+    sha256: 640347883542c06c5bff0f1a02939c0465f0ebcda83fba9097f368cec246c735
+  - folder: config
+    url: https://github.com/aivazis/config/archive/v{{ config_version }}.tar.gz
```

Comment:
What is this config project? And should it be packaged separately? That is, are you vending this package?

Reply:
We are not planning to distribute "config" via conda. It is needed to build pyre - we only stage it for building. None of its components are packaged for distribution.

Reply:
What is config, though? Does it have releases?

Reply:
"config" is a build system maintained by the developer of "pyre" - https://github.com/aivazis/config . We have convinced him to make releases that are in sync with releases of pyre. Whenever, we make an update to pyre conda package - we will work with him to use the right release of config.

Reply:
Developer of config and pyre - @aivazis is also listed as a package maintainer.

Reply:
Why isn't config just part of pyre then?

Reply:
There exist other custom projects that use config - that arent currently in the public domain but maybe one day. This would be a question for @aivazis.

Reply:
if config is usable for other projects, shouldn't it be a conda-forge package then?

Reply:
I agree. There may be others who may want to sign up to maintain the config package. We are customers of pyre and once pyre is installed, we are able to use it with other build systems like cmake. 

If someone volunteers to maintain config, we will happily use it as a dependency in our recipe.  There are numerous other tweaks that are needed to make config work within conda environment  (see build.sh)- I'm not sure the team that proposed to maintain the pyre package is up for maintaining a conda package for config.

Reply:
OK. I fully understand not wanting to maintain a conda-forge package.  However it is againts our current policy to allow external deps in general. See https://github.com/conda-forge/staged-recipes/#6-do-all-of-my-packages-dependencies-have-to-be-in-conda-forge-already

Reply:
@scopatz Thanks for the clarification. We will discuss this with @aivazis  and get back on this. Could we leave this PR open for now?

Reply:
Yes! No problem with leaving this open as long as needed